.github/workflows/build.yml

```
@@ -43,7 +43,7 @@ jobs:
     # The build job runs on the specified operating system.
     runs-on: ${{ matrix.os }}
     env:
-      TARGET_NAME: thyra-plugin-wallet_${{ matrix.target }}-${{ matrix.arch }}
+      TARGET_NAME: wallet-plugin_${{ matrix.target }}-${{ matrix.arch }}
```

Is it possible keep the dashed mode in the all sentence!?

```suggestion
      TARGET_NAME: wallet-plugin-${{ matrix.target }}-${{ matrix.arch }}
```

---

Forget... notice that is the file name in the end!